webdriver/tests/classic/web_extensions/resources/extension/background.js

```diff
@@ -0,0 +1 @@
+console.log("Hello, world!");
```

Comment:
This is minor, but it might be better to use the `browser.test.log` method.
```suggestion
browser.test.log("Hello, world!");
```